processors/npzProcessor.py

```diff
@@ -0,0 +1,295 @@
+                        dPhiMinj = utl.deltaPhi(jetPhi,metPhi).min()
+                        dEtaj12 = utl.deltaEta_vec(j1_eta,j2_eta)
+                        deltaR12j = utl.delta_R(j1_eta,j2_eta,j1_phi,j2_phi)
+
+                        # AK8 Jet Variables
+                        jetAK8pT = fjets.pt
+                        jetAK8Phi = fjets.phi
+                        jetAK8Eta = fjets.eta
+                        jetAK8M = fjets.mass
+                        j1_etaAK8 = utl.jetVar_vec(jetAK8Eta,0)
+                        j2_etaAK8 = utl.jetVar_vec(jetAK8Eta,1)
+                        j1_phiAK8 = utl.jetVar_vec(jetAK8Phi,0)
+                        j2_phiAK8 = utl.jetVar_vec(jetAK8Phi,1)
+                        dPhij1AK8 = utl.deltaPhiji_vec(j1_phiAK8,metPhi)
+                        dPhij2AK8 = utl.deltaPhiji_vec(j2_phiAK8,metPhi)
+                        dPhij1rdPhij2AK8 = utl.divide_vec(dPhij1AK8,dPhij2AK8)
+                        dPhiMinjAK8 = utl.deltaPhi(jetAK8Phi,metPhi).min()
+                        dEtaj12AK8 = utl.deltaEta_vec(j1_etaAK8,j2_etaAK8)
+                        deltaR12jAK8 = utl.delta_R(j1_etaAK8,j2_etaAK8,j1_phiAK8,j2_phiAK8)
+
+                        tau1 = fjets.tau1
+                        tau2 = fjets.tau2
+                        tau3 = fjets.tau3
+                        J_tau21 = utl.divide_vec(tau2.flatten(),tau1.flatten())
+                        J_tau32 = utl.divide_vec(tau3.flatten(),tau2.flatten())
+                        J1_tau21 = utl.tauRatio(tau2,tau1,0)
+                        J1_tau32 = utl.tauRatio(tau3,tau2,0)
+                        J2_tau21 = utl.tauRatio(tau2,tau1,1)
+                        J2_tau32 = utl.tauRatio(tau3,tau2,1)
+                        # Getting subset of variables based on number of AK8 jets
+                        # calculating variables
+                        output['evtw'] += col_accumulator(evtw)
+                        output['jw'] += col_accumulator(ak.flatten(jw))
+                        output['fjw'] += col_accumulator(ak.flatten(fjw))
+                        output['njets'] += col_accumulator(jets.counts)
+                        output['njetsAK8'] += col_accumulator(fjets.counts)
+                        output['nb'] += col_accumulator(nBJets)
+                        output['nl'] += col_accumulator((electrons.counts + muons.counts))
+                        output['ht'] += col_accumulator(ht)
+                        output['st'] += col_accumulator(st)
+                        output['met'] += col_accumulator(met)
+                        output['madHT'] += col_accumulator(madHT)
+                        output['jPt'] += col_accumulator(jets.pt.flatten())
+                        output['jEta'] += col_accumulator(jetEta.flatten())
+                        output['jPhi'] += col_accumulator(jetPhi.flatten())
+                        output['jAxismajor'] += col_accumulator(jets.axismajor.flatten())
```

Comment:
it would be nice to have a separate python file that specifies how to get each input variable from the event, to ensure that training and inference are done consistently